app/screens/options_modal/options_modal_list.js

```diff
@@ -87,6 +88,20 @@ export default class OptionsModalList extends PureComponent {
             );
         });
 
+        const cancel = (
```

Comment:
What is this? On Android it wasn't showing the delete option, how does this solve it?

Reply:
Part of cleaning this up I removed the "Cancel" button but didn't pick up on that there were a few places in the codebase that were sending a cancel button as well, but with the index for the cancel button which actually removed it on the assumption that the modal would add one at the bottom. 

I'm not entirely sure where / how this came about; but just reverting it to the state it was before.